lib/html/template/rose-bush/index.html

```diff
@@ -12,6 +12,9 @@
 <body>
 <div class="container-fluid">
 <div class="hero-unit">
+{% if logo -%}
+<img {{logo}} />
```

Comment:
I note in your example below that you configure the alt text as:
```
 [rose-bush]
logo=src="rose-favicon.png" alt="Rose Logo"
```
Not really sure that this should be something that should be configured in the logo config setting.